src/versionbits.h

```diff
@@ -16,39 +17,47 @@ static const int32_t VERSIONBITS_TOP_BITS = 0x20000000UL;
 static const int32_t VERSIONBITS_TOP_MASK = 0xE0000000UL;
 /** Total bits available for versionbits */
 static const int32_t VERSIONBITS_NUM_BITS = 29;
+/** Size of window to use for assessing warning of unknown bits */
+static const int BIT_WARNING_WINDOW = 100;
+/** Threshold to use for assessing warning of unknown bits */
+static const int BIT_WARNING_THRESHOLD = 50;
 
+// bip135: assigned numbers to these enum values
 enum ThresholdState {
-    THRESHOLD_DEFINED,
-    THRESHOLD_STARTED,
-    THRESHOLD_LOCKED_IN,
-    THRESHOLD_ACTIVE,
-    THRESHOLD_FAILED,
+    THRESHOLD_DEFINED = 0,
+    THRESHOLD_STARTED = 1,
+    THRESHOLD_LOCKED_IN = 2,
+    THRESHOLD_ACTIVE = 3,
+    THRESHOLD_FAILED = 4,
 };
 
-// A map that gives the state for blocks whose height is a multiple of Period().
-// The map is indexed by the block's parent, however, so all keys in the map
-// will either be NULL or a block with (height + 1) % Period() == 0.
+// A map that gives the state for blocks.
+// The map is indexed by the block's parent.
```

Comment:
As far as I can tell this is still caches only blocks whose height is a multiple of Period(), so why the comment change? Or am I missing something?